test/fixtures/mock-npm.js

```diff
@@ -224,6 +227,17 @@ class MockNpm {
     }
   }
 
+  get chalk () {
```

Comment:
@wraithgar ended up adding this method here to make tests pass after using `this.npm.chalk`, as it was undefined. It's just a copy-paste of what's in the original npm object. Is this the correct fix?

Reply:
That's ... weird. You shouldn't have had to do that.  publish and doctor tests didn't need this.

Reply:
yeah I spotted that too.. did I mess something up in setup?

```js
npm = mockNpm({
    prefix: t.testdirName,
    color: false,
    config: {
      global: false,
      'log-missing-names': false,
      json: false,
      omit: [],
    },
    flatOptions: {
      workspacesEnabled: true,
    },
    output: (str) => {
      npmOutput.push(str)
    },
  })
```